auth/api/grpc/client.go

```diff
@@ -129,7 +138,7 @@ func (client grpcClient) Authorize(ctx context.Context, req *mainflux.AuthorizeR
 	ctx, close := context.WithTimeout(ctx, client.timeout)
 	defer close()
 
-	res, err := client.authorize(ctx, authReq{Act: req.Act, Obj: req.Obj, Sub: req.Sub})
+	res, err := client.authorize(ctx, authReq{Act: req.GetAct(), Obj: req.GetObj(), Sub: req.GetSub()})
```

Comment:
One thought for the future: Authorization (gRPC) client **does not have to use Auth service**. We can use it to directly communicate with Keto - our Auth service acts as a mere proxy when it comes to policy evaluation and we can speed and simplify the process by calling Keto gRPC API directly. What do you think? For this initial version, I'd leave it as it is, but this is a remark for a general approach.

Reply:
As discussed before, we'll avoid calling Keto directly from clients for several reasons:
- Keto is currently used, but we wrap it's usage so it can be potentially changed in the future
- When something in Keto changes (version), you do not have to update every client and app out there, we can just adapt our wrapper to keep backward compatibility for example
- And so on